t/dedupe.c

```diff
@@ -495,12 +581,16 @@ static void show_stat(uint64_t nextents, uint64_t nchunks, uint64_t ndupextents)
 	perc *= 100.0;
 	printf("Fio setting: dedupe_percentage=%u\n", (int) (perc + 0.50));
 
+
+	if (compression) {
+	    bytes_to_human_readable_unit(unique_capacity, &uc_human, &unit);
+	    printf("Unique capacity %lu%s\n", uc_human, unit);
+	}
```

Comment:
Inconsistent indentation here.
```suggestion
	if (compression) {
		bytes_to_human_readable_unit(unique_capacity, &uc_human, &unit);
		printf("Unique capacity %lu%s\n", uc_human, unit);
	}
```

Reply:
fixed